catalyst-gateway/bin/src/cardano/event.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) enum ChainIndexerEvent {
     /// Event triggered when the number of current synchronization tasks changes.
     SyncTasksChanged {
         /// The current number of synchronization tasks.
-        current_sync_tasks: u16,
+        current_sync_tasks: usize,
```

Comment:
Why did this change?  It can't be more than 65535?
Its got nothing to do with pointers and doesn't need to be pointer sized.
```suggestion
        current_sync_tasks: u16,
```